### DISCUSSION TITLE
Tried to apply multiple times but with wrong documents but now I can't use the upload button anymore only the webcam.

### DISCUSSION THREAD

**Discussion by cbtk15** [community/community#112370] [GitHub Education] [answered]
### Hi everyone,

I've tried verifying multiple times being rejected due to a lack of current dates but now I have an email with my training plan. I want to upload a screenshot of this but the button for uploading files is gone now... Have I used it too many times? 

![image](https://github.com/community/community/assets/97601531/f838d348-fa5e-4bc7-a11b-70541db2fd0f)

**Comment by SebastienXia** [comment]
Even if you tried with the camera option, you can't ?

**Reply by cbtk15** [reply]
Unfortunately, it doesn't work 😒 tried so many times. The upload button appeared again but it is still not accepting anything.

**Comment by samus-aran** [comment, marked as answer]
Hi @cbtk15 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance

**Reply by cbtk15** [reply]
I thought that might be the case. Thank you so much! I've tried so many times... I can try again later. Thank you. I will take on your advice.

**Comment by Brickerhaus** [comment]
I am trying to take a picture but the picture says it is too large.  How do I make the picture smaller if all I can do is take a picture with my webcam or my phone when I have no access to change the picture?

**Reply by Brickerhaus** [reply]
Tried again and it required a picture of why I am taking online only courses.  I live in Ohio, and my school is in Arkansas.  So I took a picture of my drivers license I guess.  Not sure how I am supposed to show that I am a distance learner.  Autocad wasn't this hard.  Tried it again after it was rejected.  All I can get is an unofficial transcript.  And all I can do is take a picture of the screen, since my printer is broken.  If I could upload the PDF maybe this would be better.  I authenticated through my school email.  I have two factor authentication on.  I took a picture of my drivers license which has my full name that was on the unofficial transcript.  This is a bit ridiculous.
